pkg/pipelines/eventlisteners/eventlisteners.go

```diff
@@ -0,0 +1,100 @@
+package eventlisteners
+
+import (
+	triggersv1 "github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Filters for interceptors
+const (
+	devCIBuildFilters = "(header.match('X-GitHub-Event', 'pull_request') && body.action == 'opened' || body.action == 'synchronize') && body.pull_request.head.repo.full_name == 'GITHUB_REPO'"
+
+	decCDDeployFilters = "(header.match('X-GitHub-Event', 'push') && body.repository.full_name == 'GITHUB_REPO') && body.ref.startsWith('refs/heads/master')"
+
+	stageCIDryRunFilters = "(header.match('X-GitHub-Event', 'pull_request') && body.action == 'opened' || body.action == 'synchronize') && body.pull_request.head.repo.full_name == 'GITHUB_STAGE_REPO'"
+
+	stageCDDeployFilters = "(header.match('X-GitHub-Event', 'push') && body.repository.full_name == 'GITHUB_STAGE_REPO') && body.ref.startsWith('refs/heads/master')"
+)
```

Comment:
The strings that look like this `GITHUB_REPO` and `GITHUB_STAGE_REPO` need to be changed to reflect names generated based on the baseRepo parameter to the `pipelines.Bootstrap` function.

https://github.com/rhd-gitops-example/deployment-pipelines/blob/master/bootstrap.sh#L30-L31

Reply:
@bigkevmcd PTAL. Now `GITHUB_REPO` and `GITHUB_STAGE_REPO` will be replaced by baseRepo parameter.